src/index.html

```diff
@@ -10,13 +10,85 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="src/images">
+        <img
+          src="https://github.com/ArtemDanilchuk/layout_moyo-header/blob/master/src/images/logo.png?raw=true"
+          alt="Moyo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <a
+          class="nav_link is-active"
+          href="src/images"
+        >
+          APPLE
+        </a>
+        <a
+          class="nav_link"
+          href="src/images"
+        >
+          SAMSUNG
+        </a>
+        <a
+          class="nav_link"
+          href="src/images"
+        >
+          Smartphones
+        </a>
+        <a
+          class="nav_link nav_link4"
+          href="src/images"
+          data-qa="hover"
+        >
+          Laptops & Computers
+        </a>
+        <a
+          class="nav_link"
+          href="src/images"
+        >
+          Gadgets
+        </a>
+        <a
+          class="nav_link"
+          href="src/images"
+        >
+          Tablets
+        </a>
+        <a
+          class="nav_link"
+          href="src/images"
+        >
+          Photo
+        </a>
+        <a
+          class="nav_link"
+          href="src/images"
```

Comment:
The `href` attribute in this anchor tag is pointing to 'src/images'. This seems incorrect for a navigation link. Ensure that the `href` points to the correct URL or path for the 'Video' link.